### ISSUE TITLE
Error message in app creation should include public pools

### ISSUE THREAD

**Issue by ggarnier** [tsuru/tsuru#1782]
When I try to create an app and forget the `--pool` flag, the error message shows a list of pools I can choose. Currently, this message only shows pools my team has access to, but it should also show public pools.

The error message is:

```
Error: you have access to "pool1","pool2" pools. Please choose one in app creation
```

**Comment by rodrigo-brito** [comment]
I will get it.

**Comment by cezarsa** [comment]
Closed by #1786
